internal/indexer/postgres/manifestscanned_test.go

```diff
@@ -73,6 +75,8 @@ func Test_ManifestScanned_Success(t *testing.T) {
 	integration.Skip(t)
 	ctx, done := context.WithCancel(context.Background())
 	defer done()
+	const hash = `deafbeef`
```

Comment:
intended?

Reply:
yeah, it's just a hex string that's not interpreted by anything in this test.